recipes/seq/all/conanfile.py

```diff
@@ -0,0 +1,84 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir, replace_in_file
+from conan.tools.scm import Version
+from conan.tools.env import VirtualBuildEnv
+import os
+
+required_conan_version = ">=2.1"
+
+class PackageConan(ConanFile):
+    name = "seq"
+    description = "A collection of original C++14 STL-like containers and related tools"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Thermadiag/seq"
+    topics = ("formatting", "hashmap", "radix", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    short_paths = True
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "10",
+            "clang": "7",
+            "gcc": "7",
+            "msvc": "191",
+            "Visual Studio": "15",
+        }
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.16 <4]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["HEADER_ONLY"] = True
+        tc.generate()
+        venv = VirtualBuildEnv(self)
```

Comment:
Note that this is implicit in Conan 2

Reply:
@AbrilRBS 
Do you mean HEADER_ONLY set automatically with `header_only=True`?
I can't check it works fine.

Reply:
No, `VirtualBuildEnv` is automatically injected

Reply:
@AbrilRBS 
Thanks!
Fixed.